examples/howto/blackbox_external_likelihood_numpy.myst.md

```diff
@@ -175,70 +167,146 @@ x = np.linspace(0.0, 9.0, N)
 mtrue = 0.4  # true gradient
 ctrue = 3.0  # true y-intercept
 
-truemodel = my_model([mtrue, ctrue], x)
+truemodel = my_model(mtrue, ctrue, x)
 
 # make data
 rng = np.random.default_rng(716743)
 data = sigma * rng.normal(size=N) + truemodel
+```
 
+Now that we have some data we initialize the actual Op and try it out.
+
+```{code-cell} ipython3
 # create our Op
-logl = LogLike(my_loglike, data, x, sigma)
+loglike_op = LogLike()
+
+test_out = loglike_op(mtrue, ctrue, sigma, x, data)
+```
+
+`pytensor.dprint` prints a textual representation of a PyTensor graph.
+We can see our variable is the output of the `LogLike` Op and has a type of `pt.vector(float64, shape=(10,))`.
+We can also see the five constant inputs and their types.
+
+```{code-cell} ipython3
+pytensor.dprint(test_out, print_type=True)
+```
+
+Because all inputs are constant, we can use the handy `eval` method to evaluate the output
+
+```{code-cell} ipython3
+test_out.eval()
+```
+
+We can confirm this returns what we expect
+
+```{code-cell} ipython3
+my_loglike(mtrue, ctrue, sigma, x, data)
+```
+
+### Model definition
+
++++
+
+Now, let's use this Op to repeat the example shown above. To do this let's create some data containing a straight line with additive Gaussian noise (with a mean of zero and a standard deviation of `sigma`). For simplicity we set {class}`~pymc.Uniform` prior distributions on the gradient and y-intercept. As we've not set the `grad()` method of the Op PyMC will not be able to use the gradient-based samplers, so will fall back to using the {class}`pymc.Slice` sampler.
```

Comment:
The pymc.Slice isn't rendered as a link, but the reference is right here and should be left as is. The issue is on the PyMC api docs.